packages/website/next-sitemap.js

```diff
@@ -0,0 +1,50 @@
+// overrides and explicit exclusion of routes to sitemap and robots file
+const routes = [
+  { loc: '/', priority: 1 },
+  { loc: '/api-docs', priority: 0.8 },
+  { loc: '/faq', priority: 0.8 },
+  { loc: '/login', priority: 0.8 },
+  { loc: '/terms', priority: 0.8 },
+  { loc: '/callback', exclude: true },
+  { loc: '/files', exclude: true },
+  { loc: '/manage', exclude: true },
+  { loc: '/new-file', exclude: true },
+  { loc: '/new-key', exclude: true },
+]
+
+const hiddenRoutes = routes
+  .filter(({ exclude }) => exclude)
+  .map(({ loc }) => loc)
+
+module.exports = {
+  siteUrl: process.env.SITE_URL || 'https://nft.storage',
+  generateRobotsTxt: true,
+  priority: 0.7, // default priority
+  transform: async (config, path) => {
+    const route = routes.find(({ loc }) => loc === path)
```

Comment:
Just clarifying, if no route is found, you want to continue on instead of backing out/returning null?

Reply:
I think that makes sense; do you foresee a problem with that pattern?